site/config/api/config.json

```diff
@@ -0,0 +1,32 @@
+{
+  "hideMemberFields": [
+    "TypeMeta"
+  ],
+  "hideTypePatterns": [
+    "ParseError$",
+    "List$"
+  ],
+  "externalPackages": [
+    {
+      "typeMatchPrefix": "^k8s\\.io/apimachinery/pkg/apis/meta/v1\\.Duration$",
+      "docsURLTemplate": "https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration"
+    },
+    {
+      "typeMatchPrefix": "^k8s\\.io/(api|apimachinery/pkg/apis)/",
+      "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
```

Comment:
shall we use most recent k8s version?

Reply:
Hmmm not sure how we're going to keep this updated...or if we really need to. Do the docs usually change between versions?

Reply:
technically, new fields/types can be added that we reference in the code
at the same time, we use stable old fields.
but it's a bit strange that the link will be for 1.24, while the min k8s version officially support is 1.25

Reply:
Updated to 1.30